modules/jupyter-widget/src/widget.js

```diff
@@ -26,11 +26,6 @@ function loadCss(url) {
   document.getElementsByTagName('head')[0].appendChild(link);
 }
 
-function updateDeck(inputJSON, {jsonConverter, deckgl}) {
```

Comment:
Your new `updateDeck` takes different parameters. However, the code using this function is not changed. Is `DeckGLModel`'s update tested anywhere?

Reply:
It is normally tested in `test/modules/jupyter-widget/index.spec.js` but the tests are currently skipped.